### PR TITLE
[@kadena/transfer] Using new React UI components

### PR DESCRIPTION
**Pull request by MRVDH**
- Changed components to new react ui components.
- Fixed console warning of useLayoutEffect.
- Fixed new component dark theming.
- First implementation of stitches.

### CODE REVIEW

**Review by eileenmguo — APPROVED** [review]

**Review by nil-amrutlal-dept — APPROVED** [review]

**Review by rezaerami — APPROVED** [review]

**Review by KristinaSpasevska — APPROVED** [review]

File: packages/apps/transfer/src/pages/_document.tsx [hunk 1]
```diff
@@ -3,10 +3,13 @@ import {
   getCssText,
   globalCss,
 } from '@kadena/react-components';
+import { darkThemeClass } from '@kadena/react-ui/theme';
 
 import { Head, Html, Main, NextScript } from 'next/document';
 import React, { FC, ReactNode } from 'react';
 
+React.useLayoutEffect = React.useEffect;
```
Comment (sanderlooijenga) [inline packages/apps/transfer/src/pages/_document.tsx:11]: Could you add a comment in code about why this is done?

File: packages/apps/transfer/src/components/Global/AccountText/styles.ts [hunk 2]
```diff
@@ -29,12 +30,12 @@ export const StyledAccountContainer = styled('div', {
...
+export const StyledEyeOffIcon = styled(SystemIcon.EyeOffOutline, {
   width: '24px',
   height: '24px',
 });
 
-export const StyledEyeIcon = styled(SystemIcons.EyeOutline, {
+export const StyledEyeIcon = styled(SystemIcon.EyeOutline, {
   width: '24px',
   height: '24px',
 });
```
Comment (eileenmguo) [inline packages/apps/transfer/src/components/Global/AccountText/styles.ts:41]: I have a feeling the error might be coming from this. Could you try to use one of the predefined size props on the icon or add more sizes if necessary?

Comment (MRVDH) [reply packages/apps/transfer/src/components/Global/AccountText/styles.ts:41]: I tried things like $16 and $sm but both result in the same error.

Comment (eileenmguo) [reply packages/apps/transfer/src/components/Global/AccountText/styles.ts:41]: I meant removing these blocks completely and passing the size as a prop to the icon wherever it's used. The icons already have predefined sizes variants: sm, md, lg, xl, and heroHeader

So wherever it's being used you should be able to apply the 24px size using the `md` prop like this
```
<SystemIcon.EyeOutline size="sm" />
```

**Review by alber70g — APPROVED** [review]

File: packages/apps/transfer/src/pages/_document.tsx (hunk 1 above)
Comment (MRVDH) [reply packages/apps/transfer/src/pages/_document.tsx]: Decided to remove this, it resolves warnings but not very positive about the side effects

**Pull request merged**